### DISCUSSION TITLE
V2.0.0

### DISCUSSION THREAD

**Discussion by seaky** [seaky/nspanel_pro_tools_apk#42] [Announcements]
#### new features (see manual [Manual 2.x version](https://github.com/seaky/nspanel_pro_tools_apk/blob/main/README.md#manual-2x-version)
- code has been fully redesigned (see [Backward compatibility](https://github.com/seaky/nspanel_pro_tools_apk/blob/main/README.md#backward-compatibility))
  - a lot easier to add new capabilites
  - enables to add internal event listeners and therby make MQTT/HASS integration 
- new Preference based UI
  - a lot easier to add new UI elemens. Due to the small screen a scrollable view is much more convenient or usable
- screen always on/off feature (https://github.com/seaky/nspanel_pro_tools_apk/issues/5, https://github.com/seaky/nspanel_pro_tools_apk/issues/14)
- reboot device from app (https://github.com/seaky/nspanel_pro_tools_apk/issues/6)
- change hostname from app (https://github.com/seaky/nspanel_pro_tools_apk/issues/8)
- system display sleep time setting from app
- predefinied brightness scenarios based on lightsensor trigger events
- now automatically launched apps can wait for established wifi connection

#### bugfixes
- touch-screen reader memoryleak fixed
- request exclude app from battery optimization, helps to prevent app kill by system
- wake-on-wave can be turned off

<hr /><em>This discussion was created from the release <a href='https://github.com/seaky/nspanel_pro_tools_apk/releases/tag/'>V2.0.0</a>.</em>

**Comment by ianfretwell** [comment]
Installed on 2x panels running firmware v2.0.0 (with the v1.1 of tools).
No issues, subsequently upgraded panel firmware to v2.3.0 - still no issues.

Many thanks - great work.

**Comment by masterx1981** [comment]
Upgraded successfully from 1.1, fw updated to 2.2 (latest that i've found), and all seem ok 👍
